src/Altinn.Notifications.Persistence/Migration/v0.13/02-alter-function.sql

```diff
@@ -0,0 +1,33 @@
+CREATE OR REPLACE FUNCTION notifications.getemails_statusnew_updatestatus()
+    RETURNS TABLE(alternateid uuid, subject text, body text, fromaddress text, toaddress text, contenttype text) 
+    LANGUAGE 'plpgsql'
+    COST 100
+    VOLATILE PARALLEL UNSAFE
+    ROWS 1000
+AS $BODY$
+DECLARE
+    latest_email_timeout TIMESTAMP WITH TIME ZONE;
+BEGIN
+    SELECT emaillimittimeout INTO latest_email_timeout FROM notifications.resourcelimitlog WHERE id = (SELECT MAX(id) FROM notifications.resourcelimitlog);
```

Comment:
Maybe we should store the result of 
`SELECT MAX(id) FROM notifications.resourcelimitlog`
in a variable for use in the update?

Reply:
The use of the table has been designed so as the row will always be updated. We will not be adding new entries to the table. If table has been changed during execution of this function we would like the latest update to be what we go off of, as this hopefully points to a more updated wait time for when the resource has been refreshed.